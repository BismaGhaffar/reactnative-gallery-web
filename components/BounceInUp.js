import styled, { keyframes } from 'styled-components'
import { bounceInUp } from 'react-animations'

const bouncer = keyframes`${bounceInUp}`

const BounceInUp = styled.div`
  animation: 2s ${bouncer};
`

export default BounceInUp
