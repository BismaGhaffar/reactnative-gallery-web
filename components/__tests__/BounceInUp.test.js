import React from 'react'
import renderer from 'react-test-renderer'
import BounceInUp from '../BounceInUp'

it('BounceInUp can be created', () => {
  const comp = renderer.create(<BounceInUp />)
  expect(comp).toBeDefined()
})

it('<BounceInUp /> toMatchSnapshot', () => {
  const tree = renderer.create(<BounceInUp />).toJSON()
  expect(tree).toMatchSnapshot()
})
