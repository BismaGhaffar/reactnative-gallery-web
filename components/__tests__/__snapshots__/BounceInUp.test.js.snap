// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BounceInUp /> toMatchSnapshot 1`] = `
<div
  className="BounceInUp-s7n96d-0 lazCtI"
/>
`;
